Model/Sales/Repository/PackageRepository.php

```diff
@@ -42,6 +44,91 @@ public function getPackageType()
         return parent::getPackageType();
     }
 
+    /**
+     * @param $products
+     *
+     * @return string
+     */
+    public function selectPackageType(array $products): string
+    {
+        $packageTypes = AbstractConsignment::PACKAGE_TYPES_NAMES;
+        $package      = AbstractConsignment::PACKAGE_TYPE_PACKAGE_NAME;
+
+        if ($this->isMailboxActive() || $this->isDigitalStampActive()) {
+            foreach ($products as $product) {
+                foreach ($packageTypes as $packageType) {
+                    $fitProduct = $this->isAllProductsFitIn($product, $packageType);
+                    $this->setAllProductsFitInPackageType($fitProduct, $packageType);
+
+                    if ($packageType === AbstractConsignment::PACKAGE_TYPE_MAILBOX_NAME && $fitProduct) {
+                        $package = $this->fitInMailbox() ? AbstractConsignment::PACKAGE_TYPE_MAILBOX_NAME : AbstractConsignment::PACKAGE_TYPE_PACKAGE_NAME;
+                    }
+
+                    if ($packageType === AbstractConsignment::PACKAGE_TYPE_DIGITAL_STAMP_NAME && $fitProduct) {
+                        $package = $this->fitInDigitalStamp() ? AbstractConsignment::PACKAGE_TYPE_DIGITAL_STAMP_NAME : AbstractConsignment::PACKAGE_TYPE_PACKAGE_NAME;
+                    }
+                }
+            }
+        }
+
+        return $package;
+    }
+
+
+    /**
+     * @return bool
+     */
+    public function fitInMailbox()
+    {
+        if ($this->getCurrentCountry() !== 'NL') {
+            return false;
+        }
+
+        if ($this->isMailboxActive() === false) {
+            return false;
+        }
+
+        if ($this->isAllProductsFitInMailbox() === false) {
+            return false;
+        }
+
+        if ($this->getWeight() == false) {
+            return false;
+        }
+
+        if ($this->getMailboxProcent() < 100) {
```

Comment:
graag combineren in 1 if

Reply:
Wat mij betreft mag je ook alles in 1 if statement gooien als je dat zelf ook ok vindt, gewoon elke conditie op een nieuwe regel

Reply:
Done